app/uk/gov/hmrc/tariffclassificationfrontend/views/templates/application_template.scala.html

```diff
@@ -73,6 +73,14 @@ <h2 id=@headerId>@messages(headerId)</h2>
                 .pre-wrap {
```

Comment:
Is the html part of this file a copy/paste from the traderUI?   (It seems to be - same duplicate ids - which is good)

Reply:
yes it is